### PR TITLE
Intly 7192 bump fuse template version v1.7 cherrypick

### PR DESCRIPTION
**Pull request by laurafitzgerald**
## Additional Information

I'm changing the flow of the upgrade in this commit. …
The image stream upgrade should apply the same image stream templates as the install
One way would be to check what's in the install and copy it there.
Instead I am calling the main (install) from upgrade images and making that idempotent
The reason for that is so that the upgrade does not go stale over time if any of the templates change.
The version is pulled from the manifest file, if that's bumped it should cover install and upgrade.

Related Pr to master https://github.com/integr8ly/installation/pull/1327

### Installation Verification
[Install Logs](https://gist.github.com/laurafitzgerald/7c797bf088b3cf0f738f4e022a082f0f#file-intly-7192-install-L673-L703)

### Upgrade Verification
[Upgrade Log](https://gist.github.com/laurafitzgerald/93e1e603281828b8a43980a77f2cebb0#file-upgrade-intly7192-L105-L133)

The image below shows the change to the imagestreams 18 minutes previous (caused by the upgrade).
Screenshot 2020-04-29 at 13 23 17
The image below shows that a new deployment was triggered
Screenshot 2020-04-29 at 13 23 22
The image below shows the cause of the new deployment was an image change
Screenshot 2020-04-29 at 13 23 33

## Checklist

- [ ] Updated [Changelog](https://github.com/integr8ly/installation/blob/master/CHANGELOG.md)
- [x] Tested Installation
- [ ] Tested Uninstallation
- [x] Tested or Created follow on for Upgrade

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
ping @pawelpaszki for v1.7 verification

File: roles/fuse/tasks/upgrade_images.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
-#ToDo Implement CVE image update steps as described https://github.com/RHCloudServices/integreatly-help/blob/master/sops/cves/applying-cve-updates.md
-- debug:
-    msg: "TODO Implement me!!"
+# Upgrade templates
+- name: Uprade templates
```
Comment (jessesarn) [inline roles/fuse/tasks/upgrade_images.yml:3]:
```suggestion
- name: Upgrade templates
```

**Comment by pawelpaszki** [comment]
Looks good to me apart from Jesse's typo change suggestion.

File: roles/fuse/tasks/upgrade_images.yml (hunk 1 above)
Comment (laurafitzgerald) [reply roles/fuse/tasks/upgrade_images.yml:3]: Yes, fixed this in the other. Will fix it here now. Hate typos 😂

**Pull request merged**